src/edu/csus/ecs/pc2/core/report/JudgementReport.java

```diff
@@ -14,16 +14,15 @@
 import edu.csus.ecs.pc2.core.model.Judgement;
 
 /**
- * Print All Judgement Information.
+ * Print All Judgement Information including reject.ini file
  * 
  * @author pc2@ecs.csus.edu
- * @version $Id$
  */
 
-// $HeadURL$
-
 public class JudgementReport implements IReport {
 
+    public static final String NO_JUDGEMENT_PREFIX = "No - ";
```

Comment:
As stated in previous review comments, this does not appear to have anything to do with the stated purpose of Issue #119 or PR #120.  Further, this could have _significant_ ramifications; there have already been issues with the concept of having the string "No - " as part of a judgement.  This should not be included in this PR.

Reply:
I did not make this change.